pkg/vadmin/revive_db_vc.go

```diff
@@ -42,7 +42,7 @@ func (v *VClusterOps) ReviveDB(ctx context.Context, opts ...revivedb.Option) (ct
 	s.Make(opts...)
 
 	vcOpts := v.genReviveDBOptions(&s, certs)
-	_, err = v.VReviveDatabase(vcOpts)
+	_, _, err = v.VReviveDatabase(vcOpts)
```

Comment:
I am curious about what the `VCoordinationDatabase` is used for in vcluster.

Reply:
Ah, in the ReviveDB process. I think Le wants to save this information in the VCoordinationDatabase. With this information, he can write it into the config file